contracts/tgrade-valset/src/multitest/export_import.rs

```diff
@@ -0,0 +1,184 @@
+use crate::contract::{CONTRACT_NAME, CONTRACT_VERSION};
+use crate::msg::OperatorResponse;
+use crate::multitest::helpers::addr_to_pubkey;
+use crate::multitest::suite::SuiteBuilder;
+use crate::state::{
+    Config, EpochInfo, SlashingResponse, StartHeightResponse, ValidatorInfo, ValidatorSlashing,
+    ValsetState,
+};
+use cosmwasm_std::{coin, Addr, Decimal};
+use cw2::ContractVersion;
+use tg4::Tg4Contract;
+use tg_utils::Duration;
+
+#[test]
+fn export_works() {
+    let mut suite = SuiteBuilder::new()
+        .with_max_validators(6)
+        .with_min_points(3)
+        .with_operators(&["member1"])
+        .build();
+
+    let exp = suite.export().unwrap();
+
+    assert_eq!(
+        exp.contract_version,
+        ContractVersion {
+            contract: CONTRACT_NAME.to_owned(),
+            version: CONTRACT_VERSION.to_owned(),
+        }
+    );
+
+    assert_eq!(
+        exp.config,
+        Config {
+            membership: Tg4Contract(suite.membership.clone()),
+            min_points: 3,
+            max_validators: 6,
+            scaling: None,
+            epoch_reward: coin(100, "usdc"),
+            fee_percentage: Default::default(),
+            auto_unjail: false,
+            double_sign_slash_ratio: Decimal::percent(50),
+            distribution_contracts: vec![],
+            validator_group: suite.validator_group.clone(),
+            verify_validators: false,
+            offline_jail_duration: Duration::new(0)
+        }
+    );
+
+    assert_eq!(
+        exp.epoch,
+        EpochInfo {
+            epoch_length: 100,
+            current_epoch: suite.epoch().unwrap().current_epoch,
+            last_update_time: 0,
+            last_update_height: 0,
+        }
+    );
+
+    assert_eq!(exp.operators.len(), 1);
+    assert_eq!(exp.operators[0].operator, "member1");
+    assert!(!exp.operators[0].active_validator);
+
+    assert!(exp.validators.is_empty());
+
+    assert!(exp.validators_start_height.is_empty());
+
+    assert!(exp.validators_slashing.is_empty());
+}
+
+#[test]
+fn import_works() {
+    let member_addr = "reallylongaddresstofit32charact1";
```

Comment:
Kind of nitpick - to be sure there are 32 characters (I've been there) and not make reviewrs count themselfs :) you could do something like this:
```rust
"x".repeat(32)
```